client/my-sites/media-library/list-plan-promo.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import identity from 'lodash/identity';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import VideoPressUpgradeNudge from 'blocks/upgrade-nudge-expanded';
+import { PLAN_PREMIUM, FEATURE_VIDEO_UPLOADS } from 'lib/plans/constants';
+
+export const MediaLibraryListPlanPromo = ( { translate } ) => (
+	<div className="media-library__videopress-nudge-container">
+		<VideoPressUpgradeNudge
```

Comment:
In another PR I saw this renaming of the upgrade nudge to the component at hand and I'm unsure how I feel about it. It's not a VideoPress component - it's a generic nudge component that we are specializing through passing props. At this point I don't have strong feelings about it, but am definitely uneasy about not calling it something like `UpgradeNudgeExpanded`

Thoughts @artpi ?

cc: @rodrigoi didn't you have something like this recently too with this component?